examples/DotNetCore/AzureFunction/FunctionApp/Function1.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using Microsoft.Extensions.Logging;
+
+namespace FunctionApp
+{
+    public static class Function1
+    {
+        static readonly MyConfig _myConfig = new MyConfig();
```

Comment:
>static readonly [](start = 8, length = 15)

Can you add a comment to explain why static.

Reply:
Yes, absolutely.